src/main/scala/resource/Resource.scala

```diff
@@ -6,14 +6,17 @@ import java.util.zip.GZIPOutputStream
 import scala.util.control.ControlThrowable
 
 /**
- * This is a type trait for types that are considered 'resources'.   These types must be opened (potentially) and
- * closed at some point.
+ * This is a type trait for types that are considered 'resources'.
+ * These types must be opened (potentially) and closed at some point.
  */
 trait Resource[R] {
   /**
-   * Opens a resource for manipulation.  Note:  If the resource is already open by definition of existence, then
+   * Opens a resource for manipulation.
+   * 
+   * *Note:* If the resource is already open by definition of existence, then
    * this method should perform a no-op (the default implementation).
    */
+  @com.github.ghik.silencer.silent
```

Comment:
Wasn't aware of this, what's it do?

Reply:
Allow to ignore compiler warning only where not relevant (there the unused parameter)